src/AspNet.Security.OAuth.Introspection/Events/AccessTokenReceivedContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace AspNet.Security.OAuth.Introspection {
+    public class AccessTokenReceivedContext : BaseIntrospectionContext {
+        public AccessTokenReceivedContext(
+            HttpContext context, 
```

Comment:
It was supposed to work like that, but it was never implemented due to debugging issues with compilation modules in ASP.NET 5.

That said, it should also help ReSharper users :smile: